src/CattyTests/Bricks/SetBrightnessBrickTests.swift

```diff
@@ -54,7 +53,7 @@ final class SetBrightnessBrickTests: AbstractBrickTest {
         let look = Look(name: "test", andPath: "test.png")
 
         do {
-            try imageData?.write(to: URL(fileURLWithPath: object.projectPath() + "images/test.png"))
+            try imageData?.write(to: URL(fileURLWithPath: object.projectPath() + "Scene 1/images/test.png"))
```

Comment:
Please do not use the hardcoded name of the scene here. I would suggest to use `project.scene.imagesPath` instead (or similar).